Aerospike plugin sometimes failing with panic.

### Relevant telegraf.conf:

```
[agent]
flush_interval = "10s"
flush_jitter = "5s"
hostname = "aerospike-1"
interval = "10s"
round_interval = true
[inputs.aerospike]
```

### System info:
Telegraf v1.6.0 (git: release-1.6 9253f6fe)
Amazon Linux AMI release 2017.09

### Steps to reproduce:

1. Just configure aerospike plugin with provided config

### Expected behavior:

Work seamlessly.

### Actual behavior:
Failing once in a week with:
```
panic: close of closed channel

goroutine 4 [running]:
github.com/aerospike/aerospike-client-go.(*Cluster).Close(0xc42035f0e0)
        /home/ubuntu/telegraf-build/src/github.com/aerospike/aerospike-client-go/cluster.go:744 +0x4f
github.com/aerospike/aerospike-client-go.(*Client).Close(0xc4203de6f0)
        /home/ubuntu/telegraf-build/src/github.com/aerospike/aerospike-client-go/client.go:108 +0x2e
github.com/aerospike/aerospike-client-go.clientFinalizer(0xc4203de6f0)
        /home/ubuntu/telegraf-build/src/github.com/aerospike/aerospike-client-go/client.go:53 +0x2b
```
### Additional info:

This problem is already fixed in upstream aerospike-client-go by PR: https://github.com/aerospike/aerospike-client-go/pull/208
To fix this problem: 
- https://github.com/influxdata/telegraf/blob/master/Godeps#L2 should be updated with v1.30.0 aerospike-client-go version - dcec4388c832b931227afe511b7f047c6b1b53ce, or even better with latest v1.32.0 - 9701404f4c60a6ea256595d24bf318f721a7e8b8

--- a/Godeps
+++ b/Godeps
@@ -1,5 +1,5 @@
 collectd.org 2ce144541b8903101fb8f1483cc0497a68798122
-github.com/aerospike/aerospike-client-go 95e1ad7791bdbca44707fedbb29be42024900d9c
+github.com/aerospike/aerospike-client-go 9701404f4c60a6ea256595d24bf318f721a7e8b8
 github.com/amir/raidman c74861fe6a7bb8ede0a010ce4485bdbb4fc4c985
 github.com/apache/thrift 4aaa92ece8503a6da9bc6701604f69acf2b99d07
 github.com/aws/aws-sdk-go c861d27d0304a79f727e9a8a4e2ac1e74602fdc0

